extras/gomock.bzl

```diff
@@ -101,7 +108,11 @@ _gomock_source = rule(
         "library": attr.label(
             doc = "The target the Go library where this source file belongs",
             providers = [GoLibrary],
-            mandatory = True,
+            mandatory = False,
+        ),
+        "importpath": attr.string(
```

Comment:
I am afraid the attribute name `importpath` would confuse Gazelle: now there are two targets with the same `importpath` value. How about calling it `source_package`?

Reply:
I think calling it `source_package` would cause confusion with `package`, maybe `source_importpath` would be a good middle ground?

Reply:
There's actually also `self_package`, which has this description:

```
self_package: the full package import path for the generated code. The purpose of this flag is to prevent import cycles in the generated code by trying to include its own package. See [mockgen's -self_package](https://github.com/golang/mock#flags) for more information.
```

Perhaps we can avoid introducing a new parameter and just use this one?

Reply:
Hmm, on second thought it's probably better to avoid overloading the parameter given it has special meaning to `mockgen` itself. Have gone with `source_importpath` for now.